core/src/main/java/org/jclouds/osgi/Bundles.java

```diff
@@ -131,6 +131,6 @@ public T apply(Class<?> in) {
    }
 
    private static Iterable<String> splitOrEmptyAndClose(InputStream in) throws IOException {
-      return Splitter.on('\n').omitEmptyStrings().split(toStringAndClose(in));
+      return Splitter.onPattern("\r?\n").omitEmptyStrings().split(toStringAndClose(in));
```

Comment:
Yep, but this should be better/safer. The reason for this is that the jclouds repo does not use system line endings - so it is remotely possible you could get a bug expecting a system line ending and getting a different one.